instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -69,6 +70,26 @@ def __call__(self, request):
         MiddlewareMixin = object
 
 
+try:
+    from django.core.handlers.asgi import ASGIRequest
```

Comment:
Better to check for version instead, an import error may not be caused necessarily by this component being missing.

Reply:
I agree, also maintains consistency with the version check we are now doing to decide the Middleware style being used.